AfsprakenRegels.md

```diff
@@ -461,31 +461,53 @@ aanduiding gedefinieerd worden.
+
+Voor elk van deze relaties zijn er extra gegevens aan de orde. Deze gegevens worden als kenmerken (eigenschappen) gemodelleerd in een «Relatieklasse» met een bepaalde kardinaliteit. Deze gegevens kunnen dan worden vastgesteld en vastgelegd wanneer er sprake is van een relatie tussen een object van «Objecttype» C naar een object van «Objecttype» D. Merk op dat deze gegevens, dit setje gegevens, voor elke relatie apart vastgesteld en vastgelegd wordt. Zo kunnen voor de relatie van c1 naar d1 andere gegevens worden vastgesteld en vastgelegd dan voor de relatie van c1 naar d2. 
+
+Opmerkingen: 
+- een relatiesoort met kardinaliteit * van objecttype A naar objecttype B en een relatieklasse met kardinaliteit * van objecttype C naar objecttype D bestaat op instantie nivea uit * losse relaties (en niet uit 1 relatie van 1 bronobject naar * doelobjecten); 
+- de gegevens over de relatie bestaan alleen zolang de relatie tussen beide objecten bestaat en zolang elk van beide objecten zelf (nog) bestaan. wanneer een van beide niet meer bestaat, bestaat de relatie niet meer en hiermee komen ook de gegevens over de relatie te vervallen. 
+
+_Uitzondering situatie - dezelfde gegevens voor meerdere relaties_
+
 Het is echter mogelijk dat dezelfde gegevens voor meerdere relaties tegelijk
-gelden. Het is dan niet mogelijk om het te modelleren als relatieklasse. Wel
-gewenst, maar het kan niet als UML-associationClass. Als deze uitzondering het
-geval is, dan worden de relatieklasses gemodelleerd als «Objecttype», met één
-inkomende relatie en één uitgaande relatie. De oorspronkelijke kardinaliteit van
-de beoogde relatieklasse wordt hierbij behouden.
+gelden. 
+
+Het is dan niet mogelijk om het te modelleren als relatieklasse. Wel gewenst, 
+maar het kan niet als UML-associationClass. 
+Als deze uitzondering het geval is, dan worden de relatieklasses gemodelleerd als «Objecttype», 
+met één inkomende relatie en één uitgaande relatie. 
+De oorspronkelijke kardinaliteit van de beoogde relatieklasse wordt hierbij behouden. 
+
+De relatie is geobjectiveerd. Elke betrokken relatie wordt in dit geval behandeld als een object met gegevens. 
 
 <pre class='example'>
 Een Perceel kan vanwege een Perceel splitsing overgaan in twee of meerdere
-andere Percelen. De ‘overgegaan in’ relatie wordt bijgehouden in een
-relatieklasse. Gegevens over de splitsing zijn voor al deze relaties gelijk.
+andere Percelen. De ‘overgegaan in’ relatie wordt bijgehouden als een
+relatieklasse met eigen (splitsing) gegevens. Bij een splitsing ontstaan * nieuwe percelen 
+en er is daarom sprake van meerdere relaties. De gegevens over de splitsing zijn echter altijd 
+voor al deze relaties gelijk. Om deze reden wordt de relatieklasse 'overgegaan in' gemodelleerd als een objecttype.  
```

Comment:
Volgens mij halen we hier twee modelleerperspectieven door elkaar.

Je kunt prima meerdere instanties van een relatieklasse hebben, ook als deze dezelfde kenmerk, waarde combinaties hebben over alle gelijksoortige relaties in een specifiek gegevensobject. Ook als de gegevens van de relatie feitelijk gaan over hetzelfde ding vanuit een ander perspectief, bijvoorbeeld dezelfde splitsing.

Echter, wanneer je een splitsing als een aanwijsbaar ding wilt beschouwen in je UoD, zodat je gemakkelijk kunt opvragen welke objecten betrokken waren bij de splitsing, dan kun je dit als volwaardig Objecttype modelleren. 

(Dit zou ik echter niet modelleren met een 1 inkomende en 1 uitgaande relatie. Maar eerder met 1 uitgaande relatie naar het object wat gesplitst wordt, en met 1 of meerdere relaties naar de nieuwe afgseplitste objecten, maar over de richting kun je discussiëren.)

Daarmee verlies je wel het relatiegegeven "overgegaanIn" tussen de objecten.

Echter, het is ook mogelijk om een combinatie van een relatie, geobjectiveerd als Objecttype en dezelfde relatie als Relatiesoort of zelfs Relatieklasse, op te nemen in hetzelfde informatiemodel. Bijvoorbeeld een objecttype Splitsing èn een relatiesoort "overgegaanIn" tussen objecttyppen van het type "Perceel" (, wel of niet met afleidingsregels).
Het is maar afhankelijk van welke informatiebehoefte je in welke gevallen wilt kunnen bedienen.

Al met al komt het volgens mij neer op een modelleerkeuze afhankelijk van de informatiebehoefte.
Het beschrijven van dergelijke overwegingen hoort volgens mij dan ook meer thuis in een bijlage of in een primer.